elpi/coq-lib.elpi

```diff
@@ -510,6 +510,7 @@ coq.term->gref (global GR) GR :- !.
 coq.term->gref (pglobal GR _) GR :- !.
 coq.term->gref (app [Hd|_]) GR :- !, coq.term->gref Hd GR.
 coq.term->gref (let _ _ T x\x) GR :- !, coq.term->gref T GR.
+coq.term->gref (primitive (proj Proj _)) GR :- !, primitive-projection? Proj GR.
```

Comment:
```suggestion
coq.term->gref (primitive (proj Proj _)) (con C) :- !, coq.env.primitive-projection? Proj C.
```

Reply:
I am surprised I did not see any error. Is it not `const` instead of `con`?